packages/vuetify/src/components/VBtnToggle/VBtnToggle.sass

```diff
@@ -52,6 +52,9 @@
     &:after
       display: none
 
+    &:not(:first-child)
+      margin-left: -1px
```

Comment:
```suggestion
      border-left-width: 0
```

Reply:
border-width doesn't solve the problem
![image](https://user-images.githubusercontent.com/15625235/62011983-51558480-b180-11e9-90b5-e8c3bce8f6d6.png)

Reply:
@johnleider After some tests, I think border-width is the better approach. Due to the `border-width: thin` is not determined.

Change has updated.